src/services/AnalyticsService.js

```diff
@@ -0,0 +1,152 @@
+import UserService from "./UserService";
+
+export default {
+  // for tracking only when events happen, not tracking any properties
+  trackNoProperties(name, isFakeUser) {
+    if (isFakeUser) return;
+
+    window.analytics.track(name);
+  },
+
+  /* identifying a user - treats this event differently than a track event 
+  because it allows the ability to look at user specific data, and 
+  automatically attributes all following events with this user. 
+  It should be called when a new user is created, when a user is logged in, 
+  and if the parameters about the user that we are tracking are updated (we are
+  not currently tracking any user specific info that can be changed, but if
+  we were to track their school, a new identify call would have to be called
+   when profile is updated) */
+  identify(userData, isFakeUser) {
+    if (isFakeUser) return;
+
+    const listPassed = [];
+    for (var property in userData) {
+      if (userData[property] && userData[property].hasOwnProperty("passed")) {
```

Comment:
ooo this is super smart!! avoids having a hard-coded list of subject certs.

it's a ~little risky because if we add a prop later with a "passed" sub-prop but it _isn't_ a subject cert, this would accidentally count it. we'll be refactoring the whole subject schema stuff soon though so this is totally fine 👍